### ISSUE TITLE
Please upgrade Chokidar 2

### ISSUE THREAD

**Issue by thany** [storybookjs/storybook#19075]
**Describe the bug**
![image](https://user-images.githubusercontent.com/152227/187879875-d153152d-6990-40ef-b331-d2436b35e2af.png)

Chokidar 2 should probably be updated. It doesn't look good to get this message while doing a clean install.

**To Reproduce**
Do a clean install in (any?) project. Mine happens to be Vue 2.

**System**
Please paste the results of `npx sb@next info` here.
```
Need to install the following packages:
  sb@6.5.0-rc.1
Ok to proceed? (y) n
npm ERR! canceled

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\<snip>\2022-09-01T09_23_43_652Z-debug-0.log
```

**Comment by Cliabhach** [comment]
Yes, this looks a lot like #18713 and #12393 . I think the devs say it's on track for the 7.0 release? But I have no idea when that's supposed to be.

**Comment by thany** [comment]
Ohh, that'd be very nice. I'd love to know when 7.0 is scheduled for release as well. I haven't found anything like a roadmap yet, so the answer is likely "when it's ready" 😀 (at least for the time being)

**Comment by AlanGreene** [comment]
Duplicate of https://github.com/storybookjs/storybook/issues/17436 and/or the issues already mentioned.

**Comment by valentinpalkovic** [comment]
The transitive `chokidar@2` warning was from a v6-RC dependency chain. Across Storybook 7/8/9/10 the dep graph was rewritten and chokidar (where it still appears) is at v3+ via the modern builders' own watchers. Closing as obsolete.
